src/main/java/carRacing/model/Validator.java

```diff
@@ -0,0 +1,34 @@
+package carRacing.model;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class Validator {
+
+    private static final Pattern NUMERIC_PATTERN = Pattern.compile("\\d");
+    public static final int MIN_DISTANCE = 0;
+    public static final int MAX_DISTANCE = 9;
+
+
+    public void nullCheck(String input) {
```

Comment:
```suggestion
    public static final int MAX_DISTANCE = 9;

    public void nullCheck(String input) {
```
불필요한 공백 라인은 없어도 되지 않을까?